### PR TITLE
Color updates from design

### PR DESCRIPTION
**Pull request by kmurgic**
### Dependencies
No dependencies

### Documentation
https://app.asana.com/0/1201488807569337/1204411987606793/f

### Description
Design has updated some of the colors in our palette. This includes adding some new colors, removing some old colors, and modifying a few colors that already exist.

### Testing Considerations
This is fairly straightfoward in expedite-tokens, might be worthwhile for someone to double check that all the hex values in this PR match the Expedite designs here: https://www.figma.com/file/NhftgSLlaMB0GoYCZaK1LK/Expedite-v0.75?node-id=10132-85293&t=ndK4rotKRA6Jy3U4-0

This is going to be a little bit complicated to upgrade in Website since many of the colors that were removed or modified are currently being used in Website.

### Versioning
This will be a major version because it removes and changes some of our colors

### CODE REVIEW

File: CHANGELOG.md [hunk 1]
```diff
@@ -17,6 +17,15 @@ and this project adheres to [Semantic Versioning](https://semver.org/spec/v2.0.0
...
 
 ## [Unreleased]
+## [2.3.0]
+### Added
+ - Added the following colors: `pink100`, `pink200`, `pink300`, `pink500`, `pink600`, `orange100`, `orange200`, `orange300`, `orange500`, `orange600`, `royal100`, `royal200`, `royal300`, `royal500`, `royal600`, `royal700`
+
+### Changed
+  - Changed the value of the following colors: `yellow500`, `pink400`, `orange400`, `royal400`
+
+### Removed
```
Comment (BrendanJMurray) [inline CHANGELOG.md:27]: This feels like a breaking change to me so we might want to call this v3

**Review by BrendanJMurray — CHANGES_REQUESTED** [review]
I think we'll want to update the changelog and release this as a major version rather than a minor given it has breaking changes, otherwise it LGTM :+1:

File: CHANGELOG.md (hunk 1 above)
Comment (BrendanJMurray) [reply CHANGELOG.md:27]: Ah I see you did move it up to v2, but without starting over at `2.0.0` which we will want to do

Comment (kmurgic) [reply CHANGELOG.md:27]: Oh, whoops!
My bad

Comment (kmurgic) [reply CHANGELOG.md:27]: Fixed 8f45e3224bdaaf070988a998687d6c2561385054

Comment (kmurgic) [reply CHANGELOG.md:27]: I'm not sure if we up the version in `package.json` in this PR or after it goes to `master` and if we use the `npm version` command for that or if that has any unwanted side effects. This repo could really use some contributing instructions. Maybe I'll add them after I publish this.

Comment (BrendanJMurray) [reply CHANGELOG.md:27]: I'm about to run into the same issue in `niche-logger` so I feel your pain

**Review by BrendanJMurray — APPROVED** [review]

**Review by khadgis — APPROVED** [review]
LGTM!

**Pull request merged**